### DISCUSSION TITLE
What happens to user created dashboards when that user is removed from Lightdash?

### DISCUSSION THREAD

**Discussion by lenevdm** [lightdash/lightdash#9958] [Q&A] [answered]
One of our Lightdash users who has created lots of dashboards that are being used continuously by various stakeholders is leaving the organization. What happens to the dashboards he created once we remove him as a user from Lightdash? 

I'm pretty new to Lightdash so this may be a very basic question but I couldn't find an answer in the documentation.

**Comment by joaoviana** [comment, marked as answer]
Hi @lenevdm, sorry for the long wait on getting a reply, but just to clarify, when someone is removed/deleted from Lightdash, the charts/dashboards they created will **remain** in lightdash, so it won't perform a cascaded deletion. 
**However**, their syncs/scheduled deliveries will. 

Hope this answers your question!
